### ISSUE TITLE
Using Test Even in Lambda correctly runs a Glue Job and waits for it to finish but using aws invoke function locally says that a Job is already running.

### ISSUE THREAD

**Issue by sarmientoj24** [awsdocs/aws-lambda-developer-guide#245]
I have this function that runs a Glue job and waits for it to finish by checking its status every 30s and returning if it sends a state.

```
def run_job(data, layer):
    s3 = boto3.client('glue')
    job_name = JOBS[layer]
    db_affix = DEV_DB_AFFIX[data]
    dest_affix = DEV_DEST_AFFIX[data]
    partition = DEV_PARTITION[data]
    
    job_response = s3.get_job(
        JobName=job_name
    )
    
    job_arguments = job_response['Job']['DefaultArguments']
    job_arguments['--database'] = job_arguments['--database'].replace("<dev>", db_affix)
    job_arguments['--s3_dest_prefix'] = job_arguments['--s3_dest_prefix'].replace("<dev>", dest_affix)
    job_arguments['--partition'] = partition

    try:
        response = s3.start_job_run(
            JobName = job_name,
            Arguments = job_arguments
        )

        status = s3.get_job_run(JobName=job_name, RunId=response['JobRunId'])
        
        state_error = False
        if status:
            state = status['JobRun']['JobRunState']
            while state not in ['SUCCEEDED']:
                time.sleep(30)
                status = s3.get_job_run(JobName=job_name, RunId=response['JobRunId'])
                state = status['JobRun']['JobRunState']
                if state in ['STOPPED', 'FAILED', 'TIMEOUT']:
                    state_error = True
                    break
        statusCode = UNSUCCESSFUL_JOB_RUN if state_error else SUCCESSFUL_JOB_RUN
    except Exception as e:
        print(e)
        statusCode = UNSUCCESSFUL_JOB_RUN
    return statusCode
```

Whenever I run it using a test event, It finishes running the Glue Job successfully.

**But whenever I invoke a function, in my local CLI using the command below, ** it returns unsuccessfully. The error is that a Glue Job is said to be already running.  
Command
```
aws lambda invoke --function-name repopulate_dev_data --invocation-type RequestResponse --payload '{"data": "100", "layer": "T"}' response.json
{
    "StatusCode": 200,
    "ExecutedVersion": "$LATEST"
}
```

tldr, the test environment works but cmd invokation does not.

**Comment by devlinbd2** [comment]
This does not seem to be a specific documentation problem. I think you should pursue this issue in the Lambda forum.
